cmd/manager/main.go

```diff
@@ -189,3 +207,22 @@ func serveCRMetrics(cfg *rest.Config) error {
 	}
 	return nil
 }
+
+// checkIfRequiredFilesExist checks for the existence of required files and binaries before starting WMCO
+// sample error message: errors encountered with required files: could not stat /payload/hybrid-overlay.exe:
+// stat /payload/hybrid-overlay.exe: no such file or directory, could not stat /payload/wmcb.exe: stat /payload/wmcb.exe:
+// no such file or directory
+func checkIfRequiredFilesExist(requiredFiles []string) error {
+	var errors []string
+	// Iterating through file paths and checking if they are present
+	for _, file := range requiredFiles {
+		if _, err := os.Stat(file); err != nil {
+			errors = append(errors, fmt.Sprintf("could not stat %s: %s", file, err.Error()))
```

Comment:
nit: You could have just seen that file is not present and collected that file name in a slice of strings instead of creating error slice.

Reply:
There could be other reasons why `os.Stat` failed and we should ensure that we don't lose that info. Just display the filename is not sufficient.

@akhil-rane, we have been using `%v` for errors. I suggest you do that for consistency. Or does it look strange in the output?

Reply:
Ok, sorry, I should have been clear, I was thinking we should have checked for the presence of the file meaning `os.IsNotExist(err)`, if you're telling we should be much more broad, I'm fine with it.

Reply:
Nack. Please see https://github.com/openshift/windows-machine-config-operator/pull/32#discussion_r403030004

Reply:
@aravindhp changed `%s` to '%v' for errors. Output remains same.